src/main/java/com/CreditPipeline/conveyor/service/ScoringService.java

```diff
@@ -14,15 +17,60 @@ public class ScoringService {
     public boolean scoringData(ScoringDataDTO scoringDataDTO) {
 
         final long age = LocalDate.from(scoringDataDTO.getBirthdate()).until(LocalDate.now(), ChronoUnit.YEARS);
+        String empStatus = scoringDataDTO.getEmploymentDTO().getEmploymentStatus().toString();
+        Integer workExperienceTotal = scoringDataDTO.getEmploymentDTO().getWorkExperienceTotal();
+        Integer workExperienceCurrent = scoringDataDTO.getEmploymentDTO().getWorkExperienceCurrent();
 
-        if (scoringDataDTO.getEmploymentDTO().getEmploymentStatus().toString().equals(EmploymentStatus.Unemployed.toString()) ||
-                scoringDataDTO.getAmount().compareTo(scoringDataDTO.getEmploymentDTO().getSalary().multiply(BigDecimal.valueOf(20))) >= 0 ||
-                age < 20 ||
-                age > 60 ||
-                scoringDataDTO.getEmploymentDTO().getWorkExperienceTotal() < 12 ||
-                scoringDataDTO.getEmploymentDTO().getWorkExperienceCurrent() < 3) {
+        if(empStatus.equals(EmploymentStatus.Unemployed.toString())){
+            return false;
+        }
+        if (scoringDataDTO.getAmount().compareTo(scoringDataDTO.getEmploymentDTO().getSalary().multiply(BigDecimal.valueOf(20))) >= 0){
+            return false;
+        }
+        if(age < 20 || age > 60){
+            return false;
+        }
+        if(workExperienceTotal < 12 || workExperienceCurrent < 3) {
             return false;
         }
         return true;
     }
+
+    public BigDecimal calculateRate(ScoringDataDTO scoringDataDTO){
+        BigDecimal baseRate = new BigDecimal(11);
+        final long age = LocalDate.from(scoringDataDTO.getBirthdate()).until(LocalDate.now(), ChronoUnit.YEARS);
+
+        if (scoringDataDTO.getEmploymentDTO().getEmploymentStatus().toString().equals(EmploymentStatus.SelfEmployed.toString())) {
```

Comment:
У енамов в джаве есть методы сравнения, не нужно приводить к строке.
Лучше использовать сравнение в обратном порядке, так можно избежать NullPointerException
```suggestion
        if (EmploymentStatus.SelfEmployed.equals(scoringDataDTO.getEmploymentDTO().getEmploymentStatus())) {
```